ctapipe/io/containers.py

```diff
@@ -527,8 +527,19 @@ class MuonRingParameter(Container):
     ring_center_x = Field(0.0, "centre (x) of the fitted muon ring")
     ring_center_y = Field(0.0, "centre (y) of the fitted muon ring")
     ring_radius = Field(0.0, "radius of the fitted muon ring")
-    ring_phi = Field(0.0, "Orientation of fitted ring")
-    ring_inclination = Field(0.0, "Inclination of fitted ring")
+
+    @property
+    @functools.lru_cache()
```

Comment:
Does this return the cached value if I change the attributes?

Reply:
What does:

```
c = MuonRingParameter(...)
print(c.muon_ring_phi)
c.ring_center_x = 5 * u.m
print(c.muon_ring_phi)
```
return?

Reply:
Ah, I see. ...

Reply:
so maybe caching here is not so good. invalidating the cache every time an attribute is set, seems overly complex.

Reply:
also strictly speaking caching is optimizing for run time, and I did not show, that run time is an issue here, which is premature optimization.

Reply:
Of course ... in the previous version .. when somebody changed `ring_center_x` it was not guaranteed that they would also modify `ring_center_r` and `ring_center_phi` ... so having the latter two as properties seems still right, in my opinion.

Do you agree?

Reply:
Yes, but this would also effect other containers, so I would propose to handle that in another PR

Reply:
okay .. so I just renamed the container fields as you wished. (force pushed)